helpers/utils.h

```diff
@@ -52,7 +54,7 @@
 #define returnResponse(success) \
```

Comment:
I would stronlgy discourage the use of the Preprocessor for these kind of constructions. I think C++ offers templates for these kind of constructions. Templates are namespace aware and are validated by the coompiler. The returnResponse now become a replace stament for any where in the code where the term returnResponse is used, also a variable.

Reply:
Once again, as changes in helpers will touch all other plugins, sanitizing helpers module is perhaps better off taken up as a separate activity, @bobseamon?